[openusd_internal] Add missing rapidjson header file

We should upstream this patch.

--- pxr/base/js/CMakeLists.txt
+++ pxr/base/js/CMakeLists.txt
@@ -29,6 +29,7 @@ pxr_library(js
         rapidjson/filewritestream.h
         rapidjson/fwd.h
         rapidjson/internal/biginteger.h
+        rapidjson/internal/clzll.h
         rapidjson/internal/diyfp.h
         rapidjson/internal/dtoa.h
         rapidjson/internal/ieee754.h
